data/trivia/worldmap.txt

```diff
@@ -0,0 +1,196 @@
+What country is highlighted on this map? https://i.imgur.com/twTqP40.png`Ethiopia
+What country is highlighted on this map? https://i.imgur.com/qcnevI9.png`Angola
+What country is highlighted on this map? https://i.imgur.com/OrELojX.png`Cabo Verde`Cape Verde
+What country is highlighted on this map? https://i.imgur.com/qSwv9RO.png`Lesotho
+What country is highlighted on this map? https://i.imgur.com/tv8zqCC.png`Republic of Congo`Republic of the Congo
+What country is highlighted on this map? https://i.imgur.com/B2lSacI.png`Benin
+What country is highlighted on this map? https://i.imgur.com/b7k0qRD.png`Senegal
+What country is highlighted on this map? https://i.imgur.com/zRYwJsw.png`Madagascar
+What country is highlighted on this map? https://i.imgur.com/ec33YRO.png`Guinea-Bissau`Guinea Bissau
+What country is highlighted on this map? https://i.imgur.com/09dDNah.png`Egypt
+What country is highlighted on this map? https://i.imgur.com/Df9bPdT.png`Burundi
+What country is highlighted on this map? https://i.imgur.com/qqKx7hp.png`South Sudan
+What country is highlighted on this map? https://i.imgur.com/15fLQVC.png`Ghana
+What country is highlighted on this map? https://i.imgur.com/fJZatC4.png`Mozambique
+What country is highlighted on this map? https://i.imgur.com/AKegLx8.png`Sierra Leone
+What country is highlighted on this map? https://i.imgur.com/W0OhBRP.png`Tanzania
+What country is highlighted on this map? https://i.imgur.com/XqSTmsr.png`Niger
+What country is highlighted on this map? https://i.imgur.com/zKslICy.png`Cameroon
+What country is highlighted on this map? https://i.imgur.com/mWfKGSn.png`Burkina Faso
+What country is highlighted on this map? https://i.imgur.com/1GT6807.png`Guinea
+What country is highlighted on this map? https://i.imgur.com/3nivCwN.png`Mauritius
+What country is highlighted on this map? https://i.imgur.com/a3iNPku.png`The Gambia`Gambia
+What country is highlighted on this map? https://i.imgur.com/qR9uVfe.png`Liberia
+What country is highlighted on this map? https://i.imgur.com/D39yjB7.png`Tunisia
+What country is highlighted on this map? https://i.imgur.com/alsunVI.png`Equatorial Guinea
+What country is highlighted on this map? https://i.imgur.com/gr5BLLs.png`Chad
+What country is highlighted on this map? https://i.imgur.com/91UkD8N.png`Djibouti
+What country is highlighted on this map? https://i.imgur.com/wqm7dx5.png`Seychelles
+What country is highlighted on this map? https://i.imgur.com/h8nW7MC.png`Zimbabwe
+What country is highlighted on this map? https://i.imgur.com/kHtsSx9.png`Cote d'Ivoire`Ivory Coast
+What country is highlighted on this map? https://i.imgur.com/4kPlmgp.png`Rwanda
+What country is highlighted on this map? https://i.imgur.com/anbGYX6.png`Eritrea
+What country is highlighted on this map? https://i.imgur.com/Yw4rHyu.png`Namibia
+What country is highlighted on this map? https://i.imgur.com/0oN9MFH.png`Central African Republic`Car
```

Comment:
caps? CAR

Reply:
capitalization does not matter. I only consider formatting capitalization on the first answer because it is the answer the bot responds with if no one answers it correctly and it looks nicer. No one sees the other answers and they are not case sensitive